src/webhook/mutation/pod_mutator/init_container.go

```diff
@@ -58,6 +58,6 @@ func getBasePodName(pod *corev1.Pod) string {
 	return basePodName
 }
 
-func addToInitContainers(pod *corev1.Pod, initContainer *corev1.Container) {
+func attachInitContainerToPod(pod *corev1.Pod, initContainer *corev1.Container) {
```

Comment:
`attach` is not really what the function does, it just adds the init container to a list.

`inject` is what the webhook does, `attach` is mostly used when talking about doing something to something that is already running (attaching to a running container)

Reply:
```suggestion
func addInitContainerToPod(pod *corev1.Pod, initContainer *corev1.Container) {
```